web/src/widgets.ts

```diff
@@ -0,0 +1,11 @@
+import * as poll_widget from "./poll_widget";
+import * as todo_widget from "./todo_widget";
+import * as widgetize from "./widgetize";
+const zform: any = require("./zform");
+import {WidgetValue} from "./widgetize";
+
+export function initialize(): void {
+    widgetize.widgets.set("poll", poll_widget as WidgetValue);
+    widgetize.widgets.set("todo", todo_widget as WidgetValue);
+    widgetize.widgets.set("zform", zform as WidgetValue);
```

Comment:
Don’t use `as`; it’s unsafe because there’s no check at either compile time or runtime that the types actually match.